### DISCUSSION TITLE
JTBC (South Korea) PHP Parse Live Stream Problem

### DISCUSSION THREAD

**Discussion by winnie3101** [iptv-org/community#391] [Q&A] [answered]
I just finished parse JTBC Live Stream by using PHP
and my code 😉
` <?php
	/* JTBC NOW Mobile Version */ 

        $bstrURL = "https://api.jtbc.co.kr/onair/";
        $ch = curl_init();
        curl_setopt($ch, CURLOPT_URL, $bstrURL);	
	curl_setopt($ch, CURLOPT_COOKIE, "_hjSessionUser_453214=[JWT Token];");
        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
        curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
        curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, FALSE);
	curl_setopt($ch, CURLOPT_USERAGENT, 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Mobile Safari/537.36');

        $data = curl_exec($ch);
        curl_close($ch);
		
        $json = json_decode($data);
        $szPlayURL = $json->sources->HLS->HD->file;
        header("location:$szPlayURL");
?> `

When I tested and appears "403 Forbidden" and I want to help for watch live stream with VPN

Without VPN:
![image](https://user-images.githubusercontent.com/62056141/232755848-0b34aacb-9738-4bc8-b680-5a9b9186b391.png)

With VPN:
![image](https://user-images.githubusercontent.com/62056141/232755726-b997d4fe-271b-4e91-a45b-af2511bbd938.png)

Thank you ^_^

**Comment by BellezaEmporium** [comment]
Have you tried using a South Korean proxy from your PHP script ?

**Reply by winnie3101** [reply]
I haven't tried it yet
But I tried MBC Live Stream Parse and same this too

PS. No error while parsing

**Reply by BellezaEmporium** [reply]
For me it is the only option I have in mind. I cannot test VPNs or proxies where I am right now, but I guess it's most certainly an access issue from another IP than a South Korean one.

**Reply by winnie3101** [reply]
Okay, your country has blocked VPN access??

**Reply by winnie3101** [reply]
But just geoblocked

**Reply by BellezaEmporium** [reply]
> Okay, your country has blocked VPN access??

I am on a heavily restricted connection, so yes, sort of.

**Reply by winnie3101** [reply, marked as answer]
> > Okay, your country has blocked VPN access??
> 
> I am on a heavily restricted connection, so yes, sort of.

Okay, I use free VPN (vpngate.net)​
